src/Microsoft.DotNet.Build.Tasks/PackageFiles/PerfTesting.targets

```diff
@@ -3,17 +3,13 @@
 
   <!-- Perf Runner NuGet package paths -->
   <PropertyGroup Condition="'$(TargetOS)'=='Linux'">
-    <PerfDotNetCliDir>$(TestPath)../perf.dotnetcli</PerfDotNetCliDir>
+    <PerfDotNetCliDir>%24PACKAGE_DIR/perf.dotnetcli</PerfDotNetCliDir>
```

Comment:
The only way that would have been is if you accidentally did $(PACKAGE_DIR)... you have $STUFF all over in these files.  Not blocking.